src/components/squareComponent/square.spec.js

```diff
@@ -1,17 +1,90 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import GameSquare from './squareComponent';
-import Enzyme from 'enzyme';
+import Enzyme, { mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 describe('Square Component', () => {
-
   it('renders the Square component without crashing', () => {
     const gameSpaceDiv = document.createElement('div');
+
     ReactDOM.render(<GameSquare />, gameSpaceDiv);
     ReactDOM.unmountComponentAtNode(gameSpaceDiv);
+
+  });
+
+  describe('Mounting the square component', () => {
+    it('it is empty when initially mounted', () => {
```

Comment:
This test organize is very much improved from before, however you can take ever further advantage of the nesting the `it` inside the `describe. Right now, there is a bit of redundancy between the `describe` and the `it`.

I would update the code to read as follows:
```javascript
describe('Mounting the square component', () => {
    it('is empty', () => {
```

#nitpick

Reply:
Updated test code to what you suggested